web_src/css/repo.css

```diff
@@ -2417,6 +2417,10 @@
   gap: 0 !important;
 }
 
+.archived-label {
+  filter: grayscale(0.8);
```

Comment:
I think gray scale the color is unfortunate as it might collide with labels which have already a gray color. However, that's good enough for me now.

Reply:
And that's already the improved version :grin:
https://github.com/go-gitea/gitea/pull/29680#discussion_r1521804282

Reply:
We have a css class now for it, we can itterate and improve on it

Reply:
My recommendation is still `opacity: .5`, maybe `.6`.

Reply:
I'll open a new pull so we can address it here and let the merged pull stay merged ("quiet") ;)

Reply:
->  #29749